src/common/common.h

```diff
@@ -32,6 +85,18 @@ extern "C" {
         }                                                                             \
     } while (0)
 
+/**
+ * Macro for terminating the program if a given pointer is NULL.
+ * @param ptr The pointer to check.
+ * @param msg The error message to display if the pointer is NULL.
+ */
+#define LIBOQS_die(ptr, msg) do { \
```

Comment:
As far as I can tell, this macros is never used anywhere in the code.

Reply:
This can be removed as it's replaced by OQS_EXIT_IF_NULLPTR. Thanks for the catch @dstebila

Reply:
> As far as I can tell, this macros is never used anywhere in the code.
Sure. I will remove it shortly today.